roles/siege-run/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Coppied from mycourses load-generator.yml example
+- name: Configure Siege
+  template:
+    src: "files/siege.conf"
+    dest: "/.sigerc"
```

Comment:
Typo. I assume you meant `.siegerc`?

Also, what do you think about installing this in `/usr/local` somewhere instead? The root directory is not conventional.

Reply:
By default, the siege config is in root, and since we are running the configs as root, there isn't a local config file to template over. And yes, its /.siegerc, good catch.

Reply:
Okay, thanks for clarifying. If it is a Siege default, it's alright.